client/state/sharing/services/selectors.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Returns an object of service objects or null if there are no Keyring services.
+ *
+ * @param  {Object} state Global state tree
+ * @return {?Object}      Keyring services, if known.
+ */
+export function getKeyringServices( state ) {
+	return state.sharing.services.items || null;
```

Comment:
Because initial state of the reducer is an empty object, there's no way that this could ever be falsey and use the fallback value. You should remove the fallback, and perhaps also change the reducer initial state to `null`.